ledger-core/network/nds/uniproto/l2/uniserver/server.go

```diff
@@ -173,6 +178,7 @@ func (p *UnifiedServer) StartNoListen() {
 
 func (p *UnifiedServer) StartListen() {
 	if !p.ptf.HasTransports() {
+		// todo: ??? this is not explicit
```

Comment:
it is very explicit - if there are no transports yet, then we start NoListen first, before proceeding to listening transports.
please remove the comment